juju/juju
# add-storage: report IDs of added storage

## Description of change

When running the "juju add-storage" command,
report the IDs of storage added for consistency
with other commands, and so the user does not
have to match them up manually.

## QA steps

1. juju bootstrap localhost
2. juju deploy cs:~axwalk/storagetest
3. juju add-storage storagetest/0 fs
should get a message with the ID of the storage instance added ("fs/0")

do the same, but bootstrap with an older juju. should just get the storage name "fs"

## Documentation changes

Maybe need to update examples? @pmatulis 

## Bug reference

None.
